source/administration/backup.rst

```diff
@@ -1,108 +1,102 @@
 - Deploying redundant infrastructure in a separate physical data center, to protect against failures in the primary data center.
 
-A properly deployed high availability setup automatically switches over to a redundant system should a single server fail. High availability does not protect against failures such as data corruption, since errors would propagate to redundant systems.
+A properly deployed High Availability deployment automatically switches over to a redundant system should a single server fail. High Availability does not protect against failures such as data corruption, since errors would propagate to redundant systems.
 
-A "complete" disaster recovery solution would protect against both real-time hardware failures using high availability, data corruption failures using automation, and failures of the primary data center by offering both offsite backup and offsite redundant infrastructure. Because the complexity of a full disaster recovery solution is high, it is common for customers to consider trade-offs in cost and complexity relative to the anticipated risks and target recovery times.
+A "complete" disaster recovery solution would protect against both real-time hardware failures using High Availability, data corruption failures using automation, and failures of the primary data center by offering both offsite backup and offsite redundant infrastructure. Because the complexity of a full disaster recovery solution is high, it is common for customers to consider trade-offs in cost and complexity relative to the anticipated risks and target recovery times.
 
 Failover from single sign-in outage 
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-When using single sign-on with Mattermost Enterprise Edition an outage to your SSO provider can cause a partial outage on your Mattermost instance. 
+When using single sign-on with Mattermost Enterprise Edition an outage to your SSO provider can cause a partial outage on your Mattermost instance.
 
 **What happens during an SSO outage?**
 
-*Most people can still log in* - By default, when a user logs in to Mattermost they receive a session token lasting 30 days (the duration can be configured in the System Console). During an SSO outage, users with valid session tokens can continue to using Mattermost uninterrupted. 
-
-*Some people can't log in* - During an SSO outage, there are two situations under which a user cannot log in: 
-
-a) Users whose session token expires during the outage
-
-b) Users trying to log in to new devices 
+- *Most people can still log in.* By default, when a user logs in to Mattermost they receive a session token lasting 30 days (the duration can be configured in the System Console). During an SSO outage, users with valid session tokens can continue to using Mattermost uninterrupted.
+- *Some people can't log in.* During an SSO outage, there are two situations under which a user cannot log in:
```

Comment:
For some reason this sentence appears bolded. (The previous bullet point doesn't)

Reply:
It took a while but I think I fixed it. Was a really weird formatting issue - it kept bolding/italicising that point when I added nested points.